pkg/scheduler/actions/preempt/preempt.go

```diff
@@ -220,7 +219,8 @@ func preempt(
 		}
 		return nil, nil
 	}
-
+	// we should filter out those nodes that are UnschedulableAndUnresolvable status got in allocate action
+	allNodes := ssn.NodesWherePreemptionMightHelp(preemptor)
 	predicateNodes, _ := predicateHelper.PredicateNodes(preemptor, allNodes, predicateFn, true)
```

Comment:
Do we still need predict another name because `ssn.NodesWherePreemptionMightHelp` has already get the available nodes for preempt?

Reply:
Some nodes are not run predicate process in allocate action, so they will not be filter out and still need to run predicate again.